File: documentation/user/en/get-started/example/filter-order-products-by-price.cs
return evita.QueryCatalog(
	"evita",
	session => {
		return session.QueryListOfSealedEntities(
			Query(
				Collection("Product"),
				FilterBy(
					And(
						PriceInPriceLists("basic"),
						PriceInCurrency(new Currency("EUR")),
						PriceBetween(300m, null),
						EntityLocaleEquals(new CultureInfo("en"))
					)
				),
				OrderBy(
					PriceNatural(OrderDirection.Asc)
				),
				Require(EntityFetchAll())
			)
		);
	}
);

File: documentation/user/en/get-started/example/filter-order-products-by-price.java
return evita.queryCatalog(
	"evita",
	session -> {
		return session.queryListOfSealedEntities(
			query(
				collection("Product"),
				filterBy(
					and(
						priceInPriceLists("basic"),
						priceInCurrency(Currency.getInstance("EUR")),
						priceBetween(new BigDecimal(300), null),
						entityLocaleEquals(Locale.ENGLISH)
					)
				),
				orderBy(
					priceNatural(OrderDirection.ASC)
				),
				require(entityFetchAll())
			)
		);
	}
);
